lib/evaluation/evaluationClient.ts

```diff
@@ -41,11 +42,29 @@ export class EvaluationClient {
     return this._results;
   }
 
-  get results(): HandlerCheckResult[] {
+  public evaluateWrapper(pd: PresentationDefinition, vp: unknown): EvaluationResults {
+    this.evaluate(pd, vp);
+    const result: any = {};
+    result.warnings = this.results.filter((result) => result.status === Status.WARN).map((x) => JSON.stringify(x));
```

Comment:
we need a better understanding about the actual content of the warning. if string is what we want, we need to somehow extract the log into a string, not a simple string of json

Reply:
Actually we don't have any warn being generated so far, but I agree it can be sanitized to be more readable.